src/pages/RealtimeMapPage.tsx

```diff
@@ -131,8 +131,8 @@ export default function RealtimeMapPage() {
             time={to}
             onChange={(ts) => {
               const val = ts ? ts : from
-              setFrom(moment(val))
-              setTo(moment(val).add(moment(to).diff(moment(from)))) // keep the same time difference
+              setFrom(moment(val).subtract(moment(to).diff(moment(from))))
```

Comment:
I think this line should be removed
setting the `to` field should not keep the same time gap IMO

Reply:
changed my mind after seeing the preview. great work!